lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -73,3 +86,9 @@ def test_no_tab_flag_unset(self):
         response = self.client.get(self.url)
         self.assertNotIn(self.proctoring_link, response.content)
         self.assertNotIn('Allowance Section', response.content)
+
+    def has_proctoring_tab(self):
```

Comment:
as this method is doing asserts, make sure the name is depicting the behavior. Maybe name it something `AssertProctoringTabIsAvailable` and have all the asserts inside it.

Reply:
+ Docstring.

Reply:
sure. I'll change method name.